make-release.sh

```diff
@@ -30,14 +30,6 @@ bump_version () {
   BUMP_BRANCH=$2
 
   git checkout "${BUMP_BRANCH}"
-
-  echo "Updating project version to ${NEXT_VERSION}"
-  echo "${VERSION}" > VERSION
```

Comment:
instead of VERSION file, we should echo/sed the version into version.go?

Reply:
yes!